index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Exercise 2 Landing page</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] When writing comments using the inline comments feature, you should explain the reason why the code written needs to be changed and it should be easy to understand. Take a look at the following comments:

#### Example of a clear and easy-to-understand comment

 > **_Kindly_** use Stylelint linters for this project. Stylelint reports bad codes in your CSS files. It helps to enforce consistent code and prevents you from making errors in your stylesheets. 👍 😎 
 
 #### Example of a well-explained request
 
> Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

 ### Hint: You don't have to create another inline comment. You could just edit the inline comments that you have above. 💻